deno_dist/validator/validator.ts

```diff
@@ -164,6 +164,13 @@ export abstract class VBase {
     return this
   }
 
+  sanitizeValue = (value: Type) => (
```

Comment:
How about making it "private"? I think it's better that users do not use `sanitizeValue` directly. And in that case, it would be better to write this function much later. Like above the `validateRule`.